test/NuGet.Core.Tests/NuGet.PackageManagement.Test/BuildIntegration/TestPackageReferenceNuGetProject.cs

```diff
@@ -46,7 +46,7 @@ public class TestPackageReferenceNuGetProject
 
         public override string MSBuildProjectPath => PackageSpec.FilePath;
 
-        private PackageSpec PackageSpec { get; set; }
```

Comment:
Keep this private and where you need OutputPath just call GetAssetsFilePathAsync.